<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>265</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_9">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>日期时间</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="btn_readDT">
          <property name="text">
           <string>读取当前日期时间</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>字符串显示</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QTimeEdit" name="timeEdit">
          <property name="displayFormat">
           <string>H:mm:ss</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="EditTime">
          <property name="inputMask">
           <string>99:99:99;_</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="btn_bug_time">
          <property name="text">
           <string>qDebug--Time</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_setTime">
          <property name="text">
           <string>设置时间FromString</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QDateEdit" name="dateEdit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="EditDate">
          <property name="inputMask">
           <string>9999-99-99;_</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="btn_bug_date">
          <property name="text">
           <string>qDebug--Date</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_setDate">
          <property name="text">
           <string>设置日期FromString</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QDateTimeEdit" name="dateTimeEdit">
          <property name="displayFormat">
           <string>yyyy/M/d H:mm:ss</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="EditDateTime">
          <property name="inputMask">
           <string>9999-99-99 99:99:99</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QPushButton" name="btn_bug_dateTime">
          <property name="text">
           <string>qDebug--DateTime</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_setDateTime">
          <property name="text">
           <string>时间日期FromString</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>日历选择</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>选择的日期</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_4"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCalendarWidget" name="calendarWidget"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
